index.mk

```diff
@@ -250,3 +252,17 @@ grafana-push: grafana-checks
 grafana-checks:
 	@if [ -z "$(GRAFANA_API_KEY)" ]; then echo "Error: GRAFANA_API_KEY is not set" && exit 1; fi
 	@if [ -z "$(GRAFANA_DASHBOARD)" ]; then echo "Error: GRAFANA_DASHBOARD is not set" && exit 1; fi
+
+
+# Whitesource tasks
+# -----------------
+
+# Verify security and licensing of production dependencies
+whitesource:
+	@if [ -z "$(WHITESOURCE_API_KEY)" ]; then \
+		echo "Warning: WHITESOURCE_API_KEY is not set, not running whitesource"; \
+	else \
+		echo '{"apiKey": "$(WHITESOURCE_API_KEY)"}' > whitesource.config.json; \
+		whitesource run; \
```

Comment:
If whitesource is not installed this would output an error along the lines of "command not found: whitesource".